js/plot/physicalActivity.js

```diff
@@ -35,7 +35,8 @@ module.exports = function(pool, opts) {
   return function(selection) {
     opts.xScale = pool.xScale().copy();
 
-    selection.each(function(currentData) {
+    selection.each(function() {
+      var currentData = opts.data;
```

Comment:
Why this change was needed ?

Reply:
it's required to use the data passed to the component though opts, see line 262 of `chartdailyFactory.js`. 
Otherwise it would continue to use the default set of data based on the data type, physicalActivity.